item/index.html

```diff
@@ -32,6 +32,10 @@ <h1></h1>
     <a href="#">Show on map</a>
   </div>
 
+  <div class="line" id="streetview">
+    <a href="#" target="_blank">Show on Google Street View</a> <span style='font-size: smaller'>(rotation/fine positioning may be required)</span>
```

Comment:
It's not great to open things in a new window but going back to the website after you've rotated/moved in google maps is a pain.